src/components/App.tsx

```diff
@@ -84,7 +91,7 @@ const App: FC<AppProps> = ({ demo }) => {
   }, [client]);
 
   const service = useGetPathwaysService(
-    config.get(demo ? 'demoPathwaysService' : 'pathwaysService')
+    config.get(demoId === null ? 'demoPathwaysService' : 'pathwaysService')
```

Comment:
This is inconsistent with the prior behavior - `demo === true` should transform to `demoId !== null`
```suggestion
    config.get(demoId !== null ? 'demoPathwaysService' : 'pathwaysService')
```

Reply:
updated to !==